plugin/pkg/admission/security/podsecuritypolicy/admission_test.go

```diff
@@ -2205,14 +2205,11 @@ func TestPolicyAuthorizationErrors(t *testing.T) {
 	}
 	for desc, tc := range tests {
 		t.Run(desc, func(t *testing.T) {
-			var (
-				authz      = &TestAuthorizer{usernameToNamespaceToAllowedPSPs: tc.allowed}
-				privileged = true
-			)
+			authz := &TestAuthorizer{usernameToNamespaceToAllowedPSPs: tc.allowed}
 			pod := goodPod()
 			pod.Namespace = ns
 			pod.Spec.ServiceAccountName = sa
-			pod.Spec.Containers[0].SecurityContext.Privileged = &privileged
```

Comment:
why was this change needed?

Reply:
This change is not necessary. Test still passes without this change.

The motivation of this change:
It ensures that `HostPID` will only cause one validationErr for each policy, not two.

https://github.com/kubernetes/kubernetes/pull/56848/files#diff-817778329c5e0a52edffd71d550c4692R2220